### PR TITLE
ILS Fixes- 4/25/2020

### PR DESCRIPTION
**Pull request by AnthonyCK**
ZGOW (#1486) ILS 04/22

### CODE REVIEW

**Comment by AnthonyCK** [comment]
URMM (#1488) ILS 12/30
YGLA (#1485) ILS 10

**Comment by AnthonyCK** [comment]
VIDP (#1489) ILS 09

File: LOC.json [hunk 1]
```diff
@@ -39734,47 +39773,47 @@
...
-   "longitude": 77.065508,
+   "latitude": 28.53772909,
+   "longitude": 77.10947964,
    "bearing": 103.71,
    "receptionRange": 18,
    "type": 4
  },
  {
-   "name": "ILS-cat-I",
+   "name": "ILS-cat-IIIB",
```
Comment (ItsW1ll) [inline LOC.json:39809]: Please correct to III

File: LOC.json [hunk 2]
```diff
@@ -39734,47 +39773,47 @@
...
-   "longitude": 77.116896,
+   "latitude": 28.57054287,
+   "longitude": 77.08813945,
    "bearing": 271.45,
    "receptionRange": 18,
    "type": 4
  },
  {
-   "name": "ILS-cat-III",
+   "name": "ILS-cat-IIIB",
```
Comment (ItsW1ll) [inline LOC.json:39783]: Please correct to III

**Comment by AnthonyCK** [comment]
Fixed @ItsW1ll

**Comment by AnthonyCK** [comment]
KROA (#1495) LDA 06 ILS 34
DAAG (#1494) ILS 27 added ILS 09/23 fixed
VVVD (#1491) ILS 03
ENSN (#1490) LOC 19
VVCT (#1469) ILS 06

**Pull request merged**